tests/exec_path.ts

```diff
@@ -1 +1,27 @@
 console.log(Deno.execPath);
+
+async function main(): Promise<void> {
+  // See https://github.com/denoland/deno/issues/1798
+  // Assumes that tests are run from repo root
+  // Create path with "/./" segment
+  let exePath = Deno.execPath;
+  let index = exePath.indexOf("target");
+  exePath = exePath.slice(0, index) + "./" + exePath.slice(index);
+
+  const p = Deno.run({
+    args: [exePath, "eval", "console.log(Deno.execPath);"],
+    stdout: "piped"
+  });
+
+  await p.status();
+  const output = await p.output();
+  const textOutput = new TextDecoder().decode(output);
+
+  if (textOutput.indexOf("/./") > -1) {
+    throw Error("Exec path contains non-normalized components");
+  }
+}
+
+if (Deno.platform.os !== "win") {
+  main();
+}
```

Comment:
This test is pretty complicated, and I don't like that it's not runnable outside the test.py framework.. (I tried running it manually on my computer, and it failed)

This patch is a fairly minor change. I would say let's just land it without tests...